src/libs/MultiversX/backend-api.ts

```diff
@@ -190,11 +190,9 @@ export async function getOffersByIdAndNoncesFromBackendApi(
 }
 
 export async function getMarketRequirements(chainID: string): Promise<MarketplaceRequirements | undefined> {
+  const newMarketplace = new DataNftMarket(chainID === "D" ? "devnet" : "mainnet");
```

Comment:
The name "newMarketplace" is misleading. Why is it new? It's the same SC since forever